src/components/molecules/Building.vue

```diff
@@ -1,31 +1,35 @@
 <template>
-	<div>
-		<img :src="require(`/src/assets/images/${images[0].src}`)" :alt="images[0].alt" class="building-image--top">
-		<img :src="require(`/src/assets/images/${images[1].src}`)" :alt="images[1].alt" class="building-image--bottom">
+	<div class="building">
+		<img :src="require(`/src/assets/images/${images[0].src}`)" :alt="images[0].alt" class="building__image1">
+		<img :src="require(`/src/assets/images/${images[1].src}`)" :alt="images[1].alt" class="building__image2">
 	</div>	
 </template>
 
 <script>
-	import json from "/data/structure.json"
 	export default {
 		name: "Building",
-		data() { 
-			return {
-				images: json.building.images
-			}
+		props: ["images"],
+		props: {
+			images: Array
 		}
 	}
 </script>
 
 <style lang="scss">
-	.building-image {
-		width: 81%;
-		
-		&--top{
+	.building {
+    width: 100vw;
+    height: 96vw;
```

Comment:
why 96vw, in design you can get px value

Reply:
It should be scalable otherwise it doesn't make sense